.github/workflows/ci_integration.yml

```diff
@@ -146,6 +146,10 @@ jobs:
         uses: "networktocode/gh-action-setup-poetry-environment@v2"
         with:
           python-version: "${{ matrix.python-version }}"
+      - name: "Remove Installed Nautobots (Poetry Workaround)" # Workaround for https://github.com/python-poetry/poetry/issues/2918
+        run: "for i in $( seq 1 $(ls -d1 ~/.cache/pypoetry/virtualenvs/*/lib/*/site-packages/nautobot*.dist-info | wc -l) ); do echo \"$i Nautobots! Ah! Ah! Ah!\"; poetry run pip uninstall -y nautobot; done"
```

Comment:
Cute, but let's remove the Count quotes. :-)

Reply:
🤡